rest-servlet/src/main/java/org/hawkular/inventory/rest/RestEnvironments.java

```diff
@@ -57,14 +60,16 @@
 
     @GET
     @Path("/{tenantId}/environments")
-    @ApiOperation("Returns all environments under given tenant.")
+    @ApiOperation("Returns all environments under given tenant. Accepts paging query parameters.")
```

Comment:
On 28 Apr 2015, at 14:52, Lukas Krejci wrote:

>> @@ -57,14 +60,16 @@
>>
>>   @GET
>>   @Path("/{tenantId}/environments")
>> -    @ApiOperation("Returns all environments under given tenant.")
>> +    @ApiOperation("Returns all environments under given tenant. 
>> Accepts paging query parameters.")
>
> Also, how do you support multiple values passed as via one parameter 
> name (i.e. sort=a&sort=b) using `@QueryParam`?

This could be a matrix param sort=name=asc;street=desc;... and so on